src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Print.cs

```diff
@@ -25,16 +28,36 @@ internal static unsafe string DerStringToManagedString(byte[] derString)
                 return null;
             }
 
+            using (asn1String)
+            {
+                return Asn1StringToManagedString(
+                    asn1String,
+                    (bio, str, flags) => Asn1StringPrintEx(bio, str, flags));
```

Comment:
Yep, but apparently there's a language requirement that the bare `Asn1StringPrintEx` create a new delegate object each call, and the lambda is allowed to be cached.   So using the lambda on a static method is "better".